gnomad/variant_qc/random_forest.py

```diff
@@ -325,6 +325,9 @@ def test_model(
     return test_results
 
 
+from pyspark.ml.functions import vector_to_array
```

Comment:
This should be added to the top of the file with the other imports. Also, remove any imports that are no longer used after making the change below. 

I think this:
```
from pyspark.sql import SparkSession
from pyspark.sql.functions import col, udf  # pylint: disable=no-name-in-module
from pyspark.sql.types import ArrayType, DoubleType
```

Can be changed to this:
```
from pyspark.ml.functions import vector_to_array
from pyspark.sql.functions import col
```